src/internal_modules/roc_core/align_ops.h

```diff
@@ -7,7 +7,7 @@
  */
 
 //! @file roc_core/align_ops.h
-//! @brief Alignment operations.
+//! @brief TODO.
```

Comment:
Accidental change, I guess

Reply:
Fixed.

Reply:
It wasn't. But, in hindsight, I shouldn't have updated that.